flutter-studio/src/io/flutter/android/AndroidModuleLibraryManager.java

```diff
@@ -58,13 +63,28 @@ public void update() {
   }
 
   private Void updateAndroidLibraryContent(@NotNull Project androidProject) {
+    ModuleManager mgr = ModuleManager.getInstance(androidProject);
+    Module module = null;
+    for (Module mod : mgr.getModules()) {
+      if (mod.getName().equals("android") || mod.getName().equals(".android")) {
```

Comment:
wow surprising that both these module names exist.

Reply:
I agree; this is a side-effect of the Add2App project which had a reason for using ".android" vs "android".